### DISCUSSION TITLE
How to start and setup RustDesk server in openSUSE Tumbleweed?

### DISCUSSION THREAD

**Discussion by RafaelLinux** [rustdesk/rustdesk#8045] [Q&A]
I want to use, within a VPN, RustDesk to access another computer on the network, as I do with VNC. However, I'm finding it a bit more complex, so I'm asking here.

Tumbleweed has "rustdesk-server" in its repositories and I have installed it on the machine I want to access remotely. After installing that package, I list the services with "systemctl", but there is no service related to "rustdesk".

Do I have to do anything else to make the service active?

**Comment by rustdesk** [comment]
Please go to https://discord.gg/nDceKgxnkV, you may get some help. Here mainly discuss bug, nobody help you on the others.

**Comment by xlionjuan** [comment]
I recommend you build your server with docker, because the configuration of the application is easy ro replicate.
![Docker-1.png](https://github.com/rustdesk/rustdesk/assets/25243894/42dc71ad-7fe3-41fe-987b-bb2858c251bb)


According to https://en.opensuse.org/Docker

Install Docker by:
```
sudo zypper in docker docker-compose docker-compose-switch
sudo systemctl enable --now docker
```

Than follow the procedure and `docker-compose.yml` here: https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/ubuntu-server/docker/#3-setup-rustdesk-server

**Reply by RafaelLinux** [reply]
I preferred to avoid having to use containers, basically for space reasons, but I am going to consider using Podman as a manager.

Thank you very much for the information
